### PR TITLE
added tests

### PR DESCRIPTION
**Pull request by captkakao**

### CODE REVIEW

File: src/Checkers/DatabaseChecker.php [hunk 1]
```diff
@@ -4,38 +4,50 @@
...
-                isset($config['connections']) &&
-                is_array($config['connections']) &&
-                !empty($config['connections']))
-            {
-                foreach($config['connections'] as $connection) {
+                isset($this->config['connections']) &&
+                is_array($this->config['connections']) &&
+                !empty($this->config['connections'])) {
+                foreach ($this->config['connections'] as $connection) {
                     if (is_string($connection)) {
```
Comment (nbabich-dev) [inline src/Checkers/DatabaseChecker.php]: add &&

File: src/Checkers/DatabaseChecker.php [hunk 2]
```diff
@@ -4,38 +4,50 @@
...
             if (
-                isset($config['connections']) &&
-                is_array($config['connections']) &&
-                !empty($config['connections']))
-            {
-                foreach($config['connections'] as $connection) {
+                isset($this->config['connections']) &&
+                is_array($this->config['connections']) &&
+                !empty($this->config['connections'])) {
+                foreach ($this->config['connections'] as $connection) {
```
Comment (nbabich-dev) [inline src/Checkers/DatabaseChecker.php:46]: 1 free line

File: src/Checkers/DatabaseChecker.php [hunk 3]
```diff
@@ -4,38 +4,50 @@
...
                     if (is_string($connection)) {
-                        if (!DatabaseHelper::checkConnection($connection)) {
+                        if (!$this->dbHelper->checkConnection($connection)) {
                             $status = self::STATUS_FAIL;
                         }
                     }
                 }
             } else {
```
Comment (nbabich-dev) [inline src/Checkers/DatabaseChecker.php]: remove else

File: tests/Unit/ArangoCheckerTest.php [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+                    'type'     => 'arango',
+                    'host'     => '127.0.0.1',
+                    'port'     => 8529,
+                    'https'    => false,
+                    'database' => 'test',
+                    'timeout'  => 5,
+                    'query'    => 'return true',
+                ],
+                'arango_response'        => [
+                    'status_code' => \Symfony\Component\HttpFoundation\Response::HTTP_INTERNAL_SERVER_ERROR,
```
Comment (nbabich-dev) [inline tests/Unit/ArangoCheckerTest.php]: move to use

File: tests/Unit/DatabaseCheckerTest.php [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+namespace Adata\HealthChecker\Tests\Unit;
+
+use Adata\HealthChecker\Checkers\DatabaseChecker;
+use Adata\HealthChecker\Entities\HealthEntity;
+use Adata\HealthChecker\Helpers\DatabaseHelper;
+use GuzzleHttp\Client;
+use Adata\HealthChecker\Tests\TestCase;
+use ReflectionClass;
+
+class DatabaseCheckerTest extends TestCase
```
Comment (nbabich-dev) [inline tests/Unit/DatabaseCheckerTest.php:15]: phpdocs

File: tests/Unit/DatabaseCheckerTest.php [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+{
+    private $dbChecker;
```
Comment (nbabich-dev) [inline tests/Unit/DatabaseCheckerTest.php:25]: phpdocs

File: tests/Unit/ElasticsearchTest.php [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+
+class ElasticsearchTest extends TestCase
+{
+    /**
+     * @dataProvider getData
+     */
+    public function test(string $expectedHealthStatus, array $config, array $elasticResponses)
+    {
+        $responses = [];
+        foreach ($elasticResponses as $response) {
```
Comment (nbabich-dev) [inline tests/Unit/ElasticsearchTest.php:24]: 1 free line

File: src/Checkers/DatabaseChecker.php (hunk 2 above)
Comment (captkakao) [reply src/Checkers/DatabaseChecker.php:46]: free line before foreach loop?

**Pull request merged**